launch/launch/actions/execute_process.py

```diff
@@ -241,6 +242,7 @@ def __init__(
         self.__stderr_buffer = io.StringIO()
 
         self.__executed = False
+        # self.__running = False
```

Comment:
please, delete this comment before merging

Reply:
+1 to that.